src/core/session.py

```diff
@@ -133,6 +140,14 @@ def addNewFeatureToGpsTable(
 
         self._addNewFeature(feature, self._gpsTable)
 
+        stringToAdd = "{} - long:{} - lat:{}".format(
```

Comment:
The `geom` should be a "real" geometry in order to draw these transects on the map. In our case, it would be a Polyline line geometry type (list of points).